topo/node/srl/srl.go

```diff
@@ -95,6 +95,10 @@ func (n *Node) CreateNodeResource(ctx context.Context, ni node.Interface) error
 	return nil
 }
 
+func (n *Node) GetNodeResourceStatus(_ context.Context, _ node.Interface) (node.NodeStatus, error) {
+	return node.NodeStatus{}, status.Errorf(codes.Unimplemented, "Unimplemented")
```

Comment:
I wonder should this be `nil` to remove allocations in the non implemented case?

Reply:
Yes we can make it a pointer and set it to nil. I will update.